load_average/variables.tf

```diff
@@ -0,0 +1,100 @@
+variable "namespace" {
+  description = "Namespace (e.g. cp or cloudposse)."
+}
+
+variable "stage" {
+  description = "Stage (e.g. prod, dev, staging)."
+}
+
+variable "name" {
+  description = "Name (e.g. bastion or db)."
+}
+
+variable "delimiter" {
+  default = "-"
+}
+
+variable "attributes" {
+  description = "Additional attributes (e.g. policy or role)."
+  type        = "list"
+  default     = []
+}
+
+variable "tags" {
```

Comment:
I wonder if we should accept this parameter since it's not used. @comeanother thoughts?